initdb.d/04_grants.sql

```diff
@@ -129,6 +129,9 @@ CREATE ROLE download;
 GRANT USAGE ON SCHEMA sda TO download;
 GRANT SELECT ON sda.files TO download;
 GRANT SELECT ON sda.file_dataset TO download;
+GRANT SELECT ON sda.checksums TO download;
+GRANT SELECT ON sda.datasets TO download;
+GRANT SELECT ON sda.file_event_log TO download;
```

Comment:
I don't know the use case, so just guessing here - but if `file_event_log` is meant to be accessible for `download`, is that not to log usage (and it would be `INSERT` rather than `SELECT`)?

Reply:
download should only access files in the `ready` state, so it needs to read the event log to get status :)